Plugin/CppApi/include/ToolResourceProvider.h

```diff
@@ -74,6 +74,7 @@ public slots:
   void onMousePressedAndHeld(QMouseEvent& mouseEvent);
   void onMouseDoubleClicked(QMouseEvent& mouseEvent);
   void onIdentifyGraphicsOverlayCompleted(QUuid id, IdentifyGraphicsOverlayResult* identifyResult);
+  void onscreenToLocationCompleted(QUuid taskId, Point location);
```

Comment:
@khajra Sorry, missed this the first time around, but this should be `void onScreenToLocationCompleted(QUuid taskId, const Point& location);`